src/libponyrt/actor/actor.h

```diff
@@ -1,5 +1,5 @@
-#ifndef actor_h
-#define actor_h
+#ifndef actor_actor_h
```

Comment:
These I ended up "fixing" to match other headers while I was adding the windows process code, and they ended up in this branch. I can revert them to simplify the PR, but they seem more consistent this way.